src/Forms/Form/Form.stories.mdx

```diff
@@ -1,336 +1,545 @@
+	* **h** - `number`
+	* **s** - `number`
+	* **l** - `number`
+* **rgb** - `object`
+	* **a** - `number | undefined`
+	* **r** - `number`
+	* **g** - `number`
+	* **b** - `number`
+
+### Props (Input Settings)
+This component doesn't require any additional props.
+
+### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: 'color',
+			},
+			//...other fields
+		],
+	[]
+);
+```
+
+
+## FormFieldDateField
+- It allows the user to view and pick dates from a calendar widget or manually type the date in the text field.
+- [**Playground**](/?path=/story/formfields-formfielddatefield--playground)
+- Default Value: `string` - Date in UTC format transformed to string.
+
+### Props (Input Settings)
+* **showTime** - `boolean` optional - When true a time field will appear next to the date field.
+
+### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: 'date',
+				inputSettings: {
+					showTime: false,
+				}
+			},
+			//...other fields
+		],
+	[]
+);
+```
+### FormFieldDropdownSingleSelection
+- `Dropdown Single Selection` is a simple wrapper for [Material-UI Autocomplete](https://material-ui.com/components/autocomplete/#autocomplete) but with our brand colors.
+- [**Playground**](http://kube.simpleview.io:10000/?path=/docs/formfields-formfielddropdownsingleselection--playground)
+- Default Value: `string` - String of the selected options
+
+#### Props (Input Settings)
+* **placeholder** - `string` Optional - Example text shown inside of the text field portion of the dropdown.
+* **size** - `Sizes | string` Optional - Size of the field. Could be either one of the predefined sizes (please see Sizes section), or a custom size. This property can be internally overwritten if the field is placed inside of a row with other fields.
+* **options** - `array` of `object` Required - Array of options to be displayed on the options.
+	* **title** - `string` - Title of the option.
+	* **value** - `any` - Value of the option.
+
+#### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: "dropdown",
+				inputSettings: {
+					options: [
+						{ title: "The Shawshank Redemption", value: 1994 },
+						{ title: "The Godfather", value: 1972 },
+						{ title: "The Godfather: Part II", value: 1974 },
+						{ title: "The Dark Knight", value: 2008 }
+					],
+					size: "sm",
+					placeholder: "Placeholder",
+				}
+			},
+			//...other fields
+		],
+	[]
+);
+```
+---
+### FormFieldImageUpload
+- `Image Upload` This component helps upload images to the assets library, The setup supports images, videos, or documents 
+- [[**Playground**]](http://kube.simpleview.io:10000/?path=/docs/formfields-formfieldimageupload--playground)
+- Default Value: `object` - Object of the value arguments 

```

Comment:
If the default value is an object then we must add underneath all the attributes of that object so devs can know how to create that object if they decide to pass a default value. You can check FormFieldAddress' default value as inspiration.